rave-compiler/src/test/java/com/uber/rave/compiler/CompilerIntegrationTest.java

```diff
@@ -176,6 +177,33 @@ public void allInvalidIntDefModel_shouldFail() throws RaveException {
         }
     }
 
+    @Test
+    public void allValidLongDefModel_shouldSucceed() throws RaveException {
+        LongDefModel.Builder builder = new LongDefModel.Builder();
+        Collection<LongDefModel> validCases = builder.getValidCases();
+        assertFalse(validCases.isEmpty());
+        Rave rave = Rave.getInstance();
+        for (LongDefModel model : validCases) {
+            rave.validate(model);
+        }
+    }
+
+    @Test
+    public void allInvalidLongDefModel_shouldFail() throws RaveException {
+        LongDefModel.Builder builder = new LongDefModel.Builder();
+        Collection<LongDefModel> validCases = builder.getInvalidCases();
+        assertFalse(validCases.isEmpty());
+        Rave rave = Rave.getInstance();
+        for (LongDefModel model : validCases) {
+            try {
+                rave.validate(model);
+            } catch (InvalidModelException e) {
+                continue;
+            }
+            assertTrue("Model should have failed with int: " + model.getStandard() + "\n", false);
```

Comment:
Every one of these models should throw a InvalidModelException so we should never reach this line.

Reply:
Second part of my comment still stands

> Seems like what you want to do is throw an AssertionError here with that message rather than a weird assertion of a false boolean

Reply:
Yeah, I think we can improve this. I'll make the change.

Reply:
9b68ad3